process/executionTrack/result.go

```diff
@@ -1,17 +1,17 @@
 package executionTrack
 
 // CleanResult represents the outcome of a clean operation.
-type CleanResult string
+type CleanResult int
 
 const (
 	// CleanResultOK means the clean operation completed successfully.
-	CleanResultOK CleanResult = "ok"
+	CleanResultOK CleanResult = 1
```

Comment:
should this be 0?

Reply:
I started from 1 to ensure it has a distinct value from CleanResultOK in case it’s not explicitly set.